app/views/ProjectEdit/Users/UserList/index.tsx

```diff
@@ -57,44 +143,78 @@ function UserList(props: Props) {
     ] = useModalState(false);
 
     const handleModalClose = useCallback(() => {
-        setMembershipIdToEdit(undefined);
+        setProjectUserToEdit(undefined);
         setModalHidden();
     }, [setModalHidden]);
 
-    const queryForRequest = useMemo(() => ({
-        offset: (activePage - 1) * maxItemsPerPage,
-        limit: maxItemsPerPage,
-    }), [activePage]);
-    const {
-        pending: usersPending,
-        response: usersResponse,
-        retrigger: triggerGetUsers,
-    } = useRequest<MultiResponse<Membership>>({
-        url: `server://projects/${projectId}/project-memberships/`,
-        method: 'GET',
-        query: queryForRequest,
-        failureHeader: _ts('projectEdit', 'userFetchFailed'),
-    });
+    const variables = useMemo(() => ({
+        projectId,
+        page: activePage,
+        pageSize: maxItemsPerPage,
+
+    }
+    ), [projectId, activePage]);
 
     const {
-        trigger: triggerMembershipDelete,
-    } = useLazyRequest<unknown, number>({
-        url: (ctx) => `server://projects/${projectId}/project-memberships/${ctx}/`,
-        method: 'DELETE',
-        onSuccess: () => {
-            triggerGetUsers();
+        data: projectUsersResponse,
+        loading: projectUsersPending,
+        refetch,
+    } = useQuery<ProjectUsersQuery, ProjectUsersQueryVariables>(
+        PROJECT_USERS,
+        {
+            variables,
+        },
+    );
+
+    const [
+        bulkDeleteProjectMembership,
+        { loading: bulkDeleteProjectMembershipPending },
+    ] = useMutation<
+        ProjectMembershipBulkRemoveMutation,
+        ProjectMembershipBulkRemoveMutationVariables
+    >(
+        PROJECT_MEMBERSHIP_BULK_REMOVE,
+        {
+            onCompleted: (response) => {
+                if (!response?.project?.projectUserMembershipBulk) {
+                    return;
+                }
+                const {
+                    deletedResult,
+                } = response.project.projectUserMembershipBulk;
+
+                const [deletedUser] = deletedResult ?? [];
+                alert.show(
+                    `Successfully deleted ${deletedUser?.member.displayName}`,
+                    { variant: 'success' },
+                );
```

Comment:
We should check if deletion this was successful.
We should also alert if it was not.
Also, deletedUser can be undefined on line 188, which means the deletion failed.
Let's check `ok` to see if there's error